apps/modernization-ui/src/apps/search/SearchRouting.tsx

```diff
@@ -5,10 +5,12 @@ import { LaboratoryReportSearch } from './laboratory-report';
 import { InvestigationSearch } from './investigation';
 import { SimpleSearch } from './simple';
 
+const SEARCH_PAGE_SIZE = 20;
+
 const routing = [
     {
         path: 'search',
-        element: <SearchPage />,
+        element: <SearchPage paging={{ pageSize: SEARCH_PAGE_SIZE }} />,
```

Comment:
I'm trying to decide if this change belongs here or in [SearchPageProvider](https://github.com/CDCgov/NEDSS-Modernization/blob/ca26e4ce518d92a3fca50bf26d7546675f13d75b/apps/modernization-ui/src/apps/search/SearchPage.tsx#L11).  The other default value of `PageProvider` is overridden in the `SearchPageProvider`.

Reply:
I debated this myself. Let me know I can move it there.